src/actions/NamespaceAction.ts

```diff
@@ -51,6 +51,9 @@ export const fetchNamespacesIfNeeded = () => {
   return (dispatch, getState) => {
     if (shouldFetchNamespaces(getState())) {
       // Dispatch a thunk from thunk!
```

Comment:
What means this comment ?

Reply:
I think that comment should be next to the `return dispatch(asyncFetchNAmespaces(auth))`

This `NamespaceAction` was the first Redux action file introduced to our code.
That  comment is there to show that we can dispatch a `thunk` from within a `thunk`
https://github.com/reduxjs/redux-thunk#whats-a-thunk